src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -852,10 +854,12 @@ initSchema = do
         create (domainTableName Pacts)
   where
     create tablename = do
-        log "DDL" $ "initSchema: "  ++ (T.unpack . fromUtf8) tablename
-        callDb "initSchema" $ createVersionedTable tablename
+      logger <- view bdbenvLogger
+      -- TODO: should this be WARN?
```

Comment:
Unknown Pact log levels are reflected as INFO in `pactLoggers`, so that's probably what to do here.